core/src/main/java/tech/tablesaw/io/fixed/FixedWidthReadOptions.java

```diff
@@ -174,6 +173,10 @@ public Builder lineEnding(String lineEnding) {
             return this;
         }
 
+        public Builder platformLineEnding() {
```

Comment:
How about calling this `systemLineEnding` for greater consistency with the Java API?

Reply:
Done